python/ray/air/constants.py

```diff
@@ -93,10 +93,16 @@
 # as Trainable)
 DISABLE_LAZY_CHECKPOINTING_ENV = "TRAIN_DISABLE_LAZY_CHECKPOINTING"
 
+# TODO(ml-team): [Deprecation - head node syncing] Remove in 2.7.
+# Whether or not the sync-to-head behavior is enabled by default.
+# If unset, running AIR on a multi-node cluster with checkpointing will raise
+# an error telling the user to switch to cloud/NFS.
+REENABLE_DEPRECATED_SYNC_TO_HEAD_NODE = "AIR_REENABLE_DEPRECATED_SYNC_TO_HEAD_NODE"
```

Comment:
Let's prefix the env variable with `RAY_` -- I know we haven't been doing this consistently, but we really should :)